data/committees.js

```diff
@@ -290,14 +302,14 @@ export default [
 				platform: 'website',
 				link: 'https://teachla.uclaacm.com',
 			},
-			{
-				platform: 'facebook',
-				link: 'https://www.facebook.com/teachlaucla/',
-			},
 			{
 				platform: 'instagram',
 				link: 'https://www.instagram.com/acm.teachla/',
 			},
+			{
+				platform: 'facebook',
+				link: 'https://www.facebook.com/teachlaucla/',
```

Comment:
seems it's moved to https://www.facebook.com/acmteachla/

Reply:
done